doc/ongoing_operations.rst

```diff
@@ -96,11 +98,44 @@ To run a Groovy script:
  * Click on "Script Console"
  * Paste the script into that console, and click "Run"
 
-Remove a distribution / architecture
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Note: Be extra careful when deleting jobs.
+While you can easily regenerate the jobs you might lose the history of these jobs.
+
+Disable / remove a distribution / target
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you remove a distribution or target from the config the reconfigure scripts won't automatically remove the obsolete jobs.
+You can use the above Groovy script as a starting point to disable / delete them.
+
+Usually you want to disable the jobs first, wait a little bit in case you need to reenable them for another patch release, and then actually delete them.
+
+Disable all jobs related to a specific target
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Assuming that the ROS distributions is called `lunar` and the architecture is `Ubuntu Yakkety` you can disable the jobs with the following prefixes:
+* `Lsrc_uY__` which matches the Lunar source jobs for Ubuntu Yakkety.
+* `Lbin_uY64__` which matches the Lunar binary jobs for Ubuntu Yakkety for the `amd64` architecture.
+* ... add additional prefixes for other architecture.
+
+If the configuration also specifies `devel`, `doc` or `pull request` jobs for the specific target they can to be disabled too:
+* `Ldev_<key>__` which matches the Lunar devel jobs for the given build file key.
+* `Ldoc_<key>__` which matches the Lunar doc jobs for the given build file key.
+* `Lpr_<key>__` which matches the Lunar PR jobs for the given build file key.
+
+In the case of deleting the jobs the views with the same names should be empty now and can be deleted as well.
+After going to specific view you can click the "Delete *" button on the left sidebar.
+
+If your configuration also contains build files specific to the disabled target you should also disable the corresponding management jobs in the `Manage` view.
+They will start with `Ldev_<key>`, `Ldoc_<key>`, `Lrel_ <key>` followed by the key of the build file wrong your config.
```

Comment:
It may be my english but I'm not sure what this sentence means. "followed by the key of the build file wrong your config." => "build file from your config" maybe ?